ouroboros-network/src/Ouroboros/Network/BlockFetch/Decision/Common.hs

```diff
@@ -0,0 +1,677 @@
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | This module contains the part of the block fetch decisions process that is
+-- common to both the bulk sync and deadline modes.
+module Ouroboros.Network.BlockFetch.Decision.Common (
```

Comment:
Are there any changes worth reviewing in this module?

Reply:
No, everything has just been extracted from `Decision.hs` at the beginning and almost not touched since. The few changes have to do with documentation and sometimes extraction of a helper that is needed in `BulkSync.hs`.